### ISSUE TITLE
📕Proposal: remove overview tables in component docs

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/amphtml#25158]
## Overview

This is a proposal for removing the overview tables in component docs as they are hard to maintain. Instead, we should maintain the data in the frontmatter and create a better doc UX on amp.dev using this data. 

### Problem

Problems with the tables:

* hard to read on mobile
* tedious to write in markdown due to html tables
* inconsistent formatting
* hard to parse and automatically update
* required script row should be auto generated as it is potentially format specific (e.g. emails support a different version than websites).

### Proposal

Remove the tables from the component reference docs. Auto-generate content if possible, put all other information into the frontmatter.

On Github:

1. Remove examples list
2. Remove import script
3. Move status  (`PROD`, `EXPERIMENT`)  into frontmatter
4. Move supported layouts into frontmatter

On amp.dev:

1. Auto-generate and inject import script specific to the selected format.
2. Indicate status (`PROD`, `EXPERIMENT`) via a label behind the title.
3. Auto-generate and inject list of related samples.
4. Inject list of supported formats based 

### Open questions

* On Github, is it a problem if **import script** and **supported layouts** are no longer displayed in the markdown version of the component doc? (e.g. [amp-carousel](https://github.com/ampproject/amphtml/blob/master/extensions/amp-carousel/amp-carousel.md))

//cc @ampproject/wg-outreach @CrystalOnScript @pbakaus @matthiasrohmer @tharders

**Comment by caroqliu** [comment]
It looks like this issue will be fixed by #26383

**Comment by sebastianbenz** [comment]
Correct, this was the original proposal.

**Comment by sebastianbenz** [comment]
This is live.

<img width="789" alt="Screen Shot 2020-02-19 at 21 55 49" src="https://user-images.githubusercontent.com/380472/74875474-9c282b00-5362-11ea-9d71-126df3515faf.png">
